### DISCUSSION TITLE
Client Web Storage (sessionStorage & localStorage )

### DISCUSSION THREAD

**Discussion by BobSquarePants** [pallets/flask#4967] [Q&A]
Hi everyone,

I wanted to know if it's possible with Flask to set the Client Browser [Web Storage](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API) ?

Thanks.

**Comment by TheTechRobo** [comment]
I think that's doable with JavaScript only.
